lib/nrf9160_time/Kconfig

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+config NRF9160_TIME
+	bool "nRF9160 time library"
+	select AT_CMD
+	select SNTP
+
+if NRF9160_TIME
+
+config NRF9160_TIME_UPDATE_INTERVAL
+	int "Date time update interval"
+	default 3600
+
+config NRF9160_TIME_MODEM
+	bool "Get date time from the nRF9160 onboard modem"
+	default y if BOARD_NRF9160_PCA10090NS || BOARD_NRF9160_PCA20035NS
```

Comment:
This should just be

default y

I believe, although I am not sure of what the intended behaviour is here.

Reply:
The idea was to hide nrf91 specific behavior from builds concerning non-91 platforms. In seconds though it is probably better to leave it at default y and have the option opted out in the  ``prj.conf`` for the particular build.

Reply:
To hide this from non-nrf91 platforms one should adjust the depends property, not the default property.

So this should be "default y".

And we should hide this option (and others) by modifying the depends of "NRF9160_TIME", which will recursively hide these other options that depend on it already (through the pre-existing if statement).